### PR TITLE
Age Accumulation Buckets

### PR DESCRIPTION
**Pull request by Sethvg**

### CODE REVIEW

**Comment by vanshkumar** [comment]
Looks like build is breaking, could you fix that?

**Comment by vanshkumar** [comment]
Before we start combining buckets to make the custom buckets requested by MSM, it would be worthwhile to spend a bit of time figuring out why the age "16" doesn't show up in the data.. is there a bug in the parsing code somewhere? Does the data itself just not include it?

File: python/datasources/acs_poverty.py [hunk 1]
```diff
@@ -46,6 +48,44 @@
...
+def remove_suffix(input_string, suffix):
+    if suffix and input_string.endswith(suffix):
+        return input_string[: -len(suffix)]
+    return input_string
+
+
+def parseAgeString(in_str):
+    if in_str.endswith("+"):
+        return {"min": int(remove_suffix(in_str, "+")), "max": sys.maxsize}
+    else:
```
Comment (joshzarrabi) [inline python/datasources/acs_poverty.py:67]: Don't need this `else` I think

File: python/datasources/acs_poverty.py [hunk 2]
```diff
@@ -153,6 +215,33 @@ def getData(self, gcs_bucket=None):
...
+    def custom_accumulations(self):
+        new_data = {}
+
+        for data, population in self.data.items():
+            state_fip, county_fip, age, sex, race = data
+
+            above = int(population[PovertyPopulation.ABOVE])
+            below = int(population[PovertyPopulation.BELOW])
+
+            assert above != -1
```
Comment (joshzarrabi) [inline python/datasources/acs_poverty.py:227]: I don't really like the idea of having `assert` statements in our code. I would prefer checking the value and `raising` an error if it is wrong.

**Review by joshzarrabi — CHANGES_REQUESTED** [review]

**Review by joshzarrabi — APPROVED** [review]

**Pull request merged**